wfe/web-front-end.go

```diff
@@ -280,16 +279,21 @@ const (
 	malformedJWS = "Unable to read/verify body"
 )
 
-func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool) ([]byte, *jose.JsonWebKey, core.Registration, error) {
+func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool, resource core.AcmeResource) ([]byte, *jose.JsonWebKey, core.Registration, error) {
+	var err error
 	var reg core.Registration
 
 	// Read body
 	if request.Body == nil {
-		return nil, nil, reg, errors.New("No body on POST")
+		err = core.MalformedRequestError("No body on POST")
+		wfe.log.Debug(err.Error())
```

Comment:
please provide a little bit more context here. This will just spit a bunch of "No body on POST" but not say what URLs or other identifying information were involved